<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>task 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>task 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>task 3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose number of task</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="resultTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleTask1SpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>10</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100000000000000001555941612946684302426820139692106143336977058043083378116475570326498538991504744767620628086784000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="MspinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NspinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="valuespinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QLabel" name="Nlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter n:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter m:</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>The result is:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
